### ISSUE TITLE
Consider creating narrative documentation for easyaccess

### ISSUE THREAD

**Issue by ngoldbaum** [mgckind/easyaccess#165]
Some of the usage example movies in the presentation describing easyaccess are a little hard to follow. In addition, it's a little awkward to search a presentation for information and probably also not straightfoward to add new content to.

You may want to consider creating a set of sphinx documentation for easyaccess, which you could then deploy for free on readthedocs or github pages. This will produce a set of HTML documentation that might be a bit easier for your users to interact with. It would also be a good place to migrate content from the DES confluence to a public space if you care at all about that.

**Comment by mgckind** [comment]
Thanks, it's been on my TODO for some time, the presentation grew to be more than I originally planned

**Comment by mgckind** [comment]
I've going to update the current page, but want to leave this open to eventually also move to readthedocs including the docstrings from the methods inside the code, Thanks!
